scripts/release/release.js

```diff
@@ -2,25 +2,29 @@
 
 "use strict";
 
-const { exec, execSync } = require("child_process");
+const chalk = require("chalk");
+const minimist = require("minimist");
+const semver = require("semver");
+const { string: outdentString } = require("outdent");
+const { runGit, runYarn, readJson } = require("./utils");
 
 async function run() {
-  const chalk = require("chalk");
-  const { string: outdentString } = require("outdent");
-  const minimist = require("minimist");
-  const semver = require("semver");
-
-  const { readJson } = require("./utils");
+  // Fetch git tags to get the previous version number (i.e. the latest tag)
+  await runGit(["fetch", "--tags"], { cwd: __dirname });
+  // Install script's dependencies before any require
+  await runYarn(["install"], { cwd: __dirname });
```

Comment:
if dependencies of `./scripts/release` isn't installed yet, this script is failed at line:9 via requiring `./utils.js`. So I think it should be executed before executing any `require`.

Reply:
Fixed https://github.com/prettier/prettier/pull/9418/commits/82c0c83bee3dd9bae3b836f5320f6aa475ddc0fe